src/Documentation/ComponentDocs/CardDocs/CardDocs.js

```diff
@@ -0,0 +1,66 @@
+import "./CardDocs.css";
+import { Button } from "../../../MainArea/Button/Button";
+import { Image } from "../../../MainArea/Image/Image";
+import { FiPlus } from "react-icons/fi";
+import { BsCode } from "react-icons/bs";
+import Gist from "react-gist";
+export const CardDocs = () => {
+  return (
+    <div>
+      <h1>Cards</h1>
+      <p>
+        These are some of the card design available in this library. Just copy
+        the code and customize it for using it in your next project.
+      </p>
+      <h3>Usage</h3>
+      <div className="usage">
+        <div className=" card card-I">
+          <div className="card-I-title">Card Title</div>
+          <div className="card-I-subtitle">Card Subtitle Div</div>
+          <div className="card-I-text">
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s
+          </div>
+          <div className="card-I-button-div">
+            <Button
+              colorScheme="fb"
```

Comment:
"fb" is not clear, you may change it to more descriptive name

Reply:
Done correcting this. Thanks for the suggestion